src/database/upgrade/1/user-status_postgresql.sql

```diff
@@ -0,0 +1,6 @@
+# expand lastIpAddress field to support IPv6 addresses
+alter table userStatus modify column lastIpAddress char(45);
```

Comment:
This isn't the right syntax for postgresql.
```
alter table userStatusHistory alter column lastIpAddress type char(45);
```

Reply:
fixed, thanks